dockerfile.prod

```diff
@@ -58,4 +58,4 @@ EXPOSE 8000
 RUN mv ./supervisor_api.conf /etc/supervisor/conf.d
 
 # Start the supervisor
-CMD /usr/bin/supervisord -n -c /etc/supervisor/supervisord.conf
+CMD /usr/bin/supervisord -n -c /etc/supervisor/supervisord.conf
```

Comment:
It looks like the last linefeed was removed from the dockerfile.prod file - if you could put this back it would make for a cleaner pull request